configs/14.0/packages/binutils/sources

```diff
@@ -1,55 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
So there hasn't been a new binutils release since 13.0 was out?

Reply:
> So there hasn't been a new binutils release since 13.0 was out?

Right.
That said, binutils 2.35 could be released in July. I'll update my changes.